shipment.go

```diff
@@ -326,47 +323,45 @@ func (c *Client) RerateShipmentWithContext(ctx context.Context, shipmentID strin
 	return
 }
 
+// Deprecated: Use LowestShipmentRate instead.
 // LowestRate gets the lowest rate of a shipment
 func (c *Client) LowestRate(shipment *Shipment) (out Rate, err error) {
-	return c.LowestRateWithCarrier(shipment, nil)
+	return c.LowestShipmentRate(shipment)
 }
 
+// LowestShipmentRate gets the lowest rate of a shipment
+func (c *Client) LowestShipmentRate(shipment *Shipment) (out Rate, err error) {
+	return c.LowestShipmentRateWithCarrier(shipment, nil)
+}
+
+// Deprecated: Use LowestShipmentRateWithCarrier instead.
```

Comment:
Question: Have we confirmed the godoc pics up on this correctly or do we instead need to have this be the 2nd line in the summary?

Reply:
https://github.com/golang/go/wiki/Deprecated#:~:text=The%20paragraph%20does%20not%20have%20to%20be%20the%20last%20paragraph%20in%20the%20doc%20comment.